### ISSUE TITLE
[SUGGESTION]

### ISSUE THREAD

**Issue by FeelsBadMan1** [TadaNoSenshi/casino#1]
Good day,
I have discovered a few strange things in your code that shouldn't be there and I have a suggestion;
You have mixed different languages in your code so that the bot sends an English message when joining a guild but the answers of commands are mainly German. I think that's the first thing that could be reworked, as well as the following suggestion: you only use json, which can get quite messy with a certain size of guild members, let alone bot users. I would recommend you to use a database of your choice, but I recommend MongoDB.

**Comment by TadaNoSenshi** [comment]
Yes I know but I have fixed this Problems some das ago but I forgett to Update this git, thank you :)
